web-client/src/presenter/sequences/updateCaseDeadlineSequence.ts

```diff
@@ -26,7 +25,6 @@ export const updateCaseDeadlineSequence = [
       clearScreenMetadataAction,
       clearModalAction,
       clearModalStateAction,
-      getCaseDeadlinesForCaseAction,
```

Comment:
Since we write the case deadline, we use the response from that write instead of re-fetching from the DB, which can lead to an east-west issue (east updates, but west hasn't gotten the update yet).